releasenotes/notes/fix-elasticsearch-always-populate-span-tags-aa904d748362f2d1.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
nit: the filename of this release note is `fix-elasticsearch-always-populate-span-tags` but that doesn't quick match up to the contents of the release note about celery.